package.json

```diff
@@ -83,6 +83,7 @@
     "npm-snapshot": "1.0.3",
     "prettier": "1.13.4",
     "puppeteer": "1.3.0",
+    "tslint-no-circular-imports": "0.5.0",
```

Comment:
This is new to me, I have not seen this: tslint-no-circular-imports. So I guess a circular import was added and this was needed to fix this bug? Just trying to relate how this is related to the basepath not being set?

Reply:
I think you are missing Edgar’s first comment :)

Reply:
Ah ok @josejulio thanks for the clarification.

Reply:
Great, I was worried by that, I didn't understand why it wasn't loading the value.

Reply:
Wow, this was a nice catch. I didn't understand the root case as well.
Just curiosity, how was this spotted ? looking into the generated code on a debug session from the browser ?
If there is some setup for this, it can be worth to share.
Thanks !

Reply:
I saw this comment: https://github.com/babel/babel/issues/4094#issuecomment-245410649
So, that made me suspect about a cycle. In order to check order if execution, simple `console.log` statements at the beginning of each file shown that `app/History.tsx` was being run first.
So, no complex setup.

Reply:
Good catch.
I personally admit I didn't put attention into circular dependencies, for some reason I thought the compiler was already checking it.
Thanks for sharing the details @israel-hdez .